src/lib.rs

```diff
@@ -1095,6 +1144,29 @@ mod tests {
         assert_eq!(v.get(size_of::<usize>() * 8 - 2), Some(true));
     }
 
+
+    #[test]
+    fn test_from_bytes() {
+        // Example adopted from BitVec
+        let v = Vob::from_bytes(&[0b10100000, 0b00010010]);
+        assert_eq!(v, vob![true, false, true, false, false, false, false, false,
+                           false, false, false, true, false, false, true, false]);
+
+        // On a 64-bit machine, make sure we test that a complete word is dealt with correctly.
+        let v = Vob::from_bytes(&[0b10100000, 0b00010010, 0b00110101, 0b11001010,
+                                  0b00110001, 0b10010101, 0b01111100, 0b01010001]);
+        assert_eq!(v, vob![true, false, true, false, false, false, false, false,
+                           false, false, false, true, false, false, true, false,
+                           false, false, true, true, false, true, false, true,
+                           true, true, false, false, true, false, true, false,
+                           false, false, true, true, false, false, false, true,
+                           true, false, false, true, false, true, false, true,
+                           false, true, true, true, true, true, false, false,
+                           false, true, false, true, false, false, false, true]);
+
```

Comment:
blank lines, kill?

Reply:
Test `Vob::from_bytes(&[])`? (empty byte vec).

Reply:
Agreed about testing the empty slice.